crates/swc_plugin_proxy/src/memory_interop/read_returned_result_from_host.rs

```diff
@@ -1,10 +1,11 @@
 #[cfg_attr(not(target_arch = "wasm32"), allow(unused))]
 use swc_common::plugin::{
     deserialize_from_ptr, deserialize_from_ptr_into_fallible, PluginSerializedBytes,
+    VersionedSerializable,
 };
 
 /// A struct to exchange allocated data between memory spaces.
-#[derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize)]
+#[derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize, Default)]
```

Comment:
Same here

Reply:
I'll take it out with other refactorings, but this is technically possible to be correctly one, isn't it? If we'll need default later somehow, I expect this struct can use default derive.

Reply:
Problem is that this is a pointer, as `0` is definitely not a good number for pointer

Reply:
Yes, but for a new value / or default I'd expect 0 as invalid ptr number to be filled in. But in any case, that's somethign to discuss in the future and PR removed default impl.